docker/dev/Dockerfile

```diff
@@ -10,7 +10,7 @@ FROM nvidia/cuda:11.2.2-cudnn8-devel-ubuntu20.04
 # note that this image uses Python 3.8
 ENV IMG_NAME=11.2.2-cudnn8-devel-ubuntu20.04 \
     # declare the cuda version for pulling appropriate jaxlib wheel
-    JAXLIB_CUDA=112
+    JAXLIB_CUDA=111
```

Comment:
there is no longer CUDA 112 version because the 111 built will work for the remaining versions in the 11x series.

Reply:
I see, thanks for claryfying!